react/BaseContent.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { useOrderForm } from 'vtex.order-manager/OrderForm'
+import { useCssHandles } from 'vtex.css-handles'
+import { Button } from 'vtex.styleguide'
+
+import { useMinicartState } from './MinicartContext'
+
+interface OrderFormContext {
+  orderForm: OrderForm
+  loading: boolean
+}
+
+interface Props {
+  sideBarMode: boolean
+  finishShoppingButtonLink: string
+}
+
+const CSS_HANDLES = [
+  'minicartContent',
+  'minicartFooter',
+  'minicartEmptyStateText',
+  'minicartEmptyStateContainer',
+] as const
+
+const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
+  const { orderForm, loading }: OrderFormContext = useOrderForm()
+  const handles = useCssHandles(CSS_HANDLES)
+  const { variation } = useMinicartState()
+
+  const sideBarStyles = {
+    height: '100%',
+  }
+
+  const popupStyles = {
+    maxHeight: 600,
+  }
+
+  return !loading && orderForm.items.length === 0 ? (
+    <div
+      className={`${handles.minicartEmptyStateContainer} pa9 flex justify-center`}
+      style={{ width: 340 }}
+    >
+      <span className={`${handles.minicartEmptyStateText} t-body`}>
+        <FormattedMessage id="store/minicart.empty-state" />
+      </span>
+    </div>
+  ) : (
+    <div
+      className={`${handles.minicartContent} flex flex-column justify-between`}
+      style={variation === 'drawer' ? sideBarStyles : popupStyles}
+    >
+      <div className="w-100 overflow-y-scroll">
```

Comment:
Avoid using `overflow-y-scroll` because the scrollbar appears even if the content do not overflow its container. Use `overflow-y-auto`. :smile:

Reply:
Awesome! Did not know about that!